omni-relayer/src/workers/evm.rs

```diff
@@ -0,0 +1,225 @@
+                        else {
+                            return;
+                        };
+
+                        let sender = config.near.token_locker_id.clone();
+
+                        // If storage is sufficient, then flag should be false, otherwise true
+                        let sender_is_storage_deposit = !utils::storage::is_storage_sufficient(
+                            &jsonrpc_client,
+                            &token,
+                            &sender,
+                        )
+                        .await
+                        .unwrap_or_default();
+                        let recipient_is_storage_deposit = !utils::storage::is_storage_sufficient(
+                            &jsonrpc_client,
+                            &token,
+                            &recipient,
+                        )
+                        .await
+                        .unwrap_or_default();
+
+                        let fin_transfer_args = FinTransferArgs {
+                            chain_kind: ChainKind::Eth,
+                            native_fee_recipient: config.evm.relayer.clone(),
+                            storage_deposit_args: StorageDepositArgs {
+                                token,
+                                accounts: vec![
+                                    (sender, sender_is_storage_deposit),
+                                    (recipient, recipient_is_storage_deposit),
+                                ],
+                            },
+                            prover_args,
+                        };
+
+                        match connector.near_fin_transfer(fin_transfer_args).await {
+                            Ok(tx_hash) => {
+                                log::info!("Finalized InitTransfer: {:?}", tx_hash);
+                                utils::redis::remove_event(
+                                    &mut redis_connection,
+                                    utils::redis::ETH_WITHDRAW_EVENTS,
+                                    key,
+                                )
+                                .await;
+                            }
+                            Err(err) => log::error!("Failed to finalize InitTransfer: {}", err),
+                        }
+                    }
+                }));
+            }
+        }
+
+        join_all(handlers).await;
+
+        tokio::time::sleep(tokio::time::Duration::from_secs(
+            utils::redis::SLEEP_TIME_AFTER_EVENTS_PROCESS_SECS,
+        ))
+        .await;
+    }
+}
+
+pub async fn claim_native_fee(
+    redis_client: redis::Client,
+    connector: Arc<OmniConnector>,
+) -> Result<()> {
+    let redis_connection = redis_client.get_multiplexed_tokio_connection().await?;
+
+    loop {
+        let mut redis_connection_clone = redis_connection.clone();
+        let Some(events) = utils::redis::get_events(
+            &mut redis_connection_clone,
+            utils::redis::ETH_WITHDRAW_EVENTS.to_string(),
```

Comment:
Why ETH_WITHDRAW_EVENTS not SignClaimNativeFeeEvent?

Reply:
Fixed in 0f3efd2b